packages/block-editor/src/components/dimension-control/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Icon,
+	SelectControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import {
+	Fragment,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import sizesTable, { findSizeBySlug } from './sizes';
+
+export function DimensionControl( props ) {
+	const { label, icon, sizes = sizesTable, currentSize, onSpacingChange, onReset, className = '' } = props;
```

Comment:
Normally to reset the value in our components onChange is called with an undefined value. Would the same work here and remove onReset be an option?

Reply:
Good question. I'll take a look as I have a feeling there was a specific reason for it being implemented like it is.

Reply:
Ok done

Reply:
Nope. All fixed.